cnab/app/run

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+action=$CNAB_ACTION
+name=$CNAB_INSTALLATION_NAME 
+use_node_port=""
+if [ "$NODE_PORT" = true ]; then
+    use_node_port="--set gateways.istio-ingressgateway.type=NodePort"
+fi
+istio_chart=`ls /charts/istio*tgz`
+riff_chart=`ls /charts/riff*tgz`
+
+case $action in
+    install)
+        echo "Installing helm client"
+        helm init --client-only
+
+        echo "Installing Istio"
+        helm install -n istio ${istio_chart} --namespace istio-system ${use_node_port} --wait
```

Comment:
```suggestion
        helm install -n istio /charts/istio*.tgz --namespace istio-system ${use_node_port} --wait
```

Reply:
I could have sworn that helm did not like "*" in the file name, but seems to work!

Reply:
the wildcard is resolved by bash, helm never sees it